packages/admin/src/common/toolbar/breadcrumb/ToolbarBreadcrumbs.tsx

```diff
@@ -12,19 +12,34 @@ const BreadcrumbLink = React.forwardRef<HTMLAnchorElement, RouterLinkProps>(({ h
 
 export const ToolbarBreadcrumbs: React.FunctionComponent = () => {
     const themeProps = useThemeProps();
+    const classes = useStyles();
 
     return (
         <StackApiContext.Consumer>
             {(stackApi) => {
                 return (
                     <>
-                        {stackApi?.breadCrumbs.map(({ id, url, title }) => {
+                        {stackApi?.breadCrumbs.map(({ id, url, title }, index) => {
+                            const showSeparator = index < stackApi?.breadCrumbs.length - 1;
+                            const isActive = index === stackApi?.breadCrumbs.length - 1;
                             return (
-                                <ToolbarItem key={id}>
-                                    <Link to={url} component={BreadcrumbLink} color={"inherit"}>
-                                        <Typography {...themeProps.typographyProps}>{title}</Typography>
-                                    </Link>
-                                </ToolbarItem>
+                                <>
+                                    <div className={classes.item} key={id}>
+                                        <Link to={url} component={BreadcrumbLink} color={"inherit"}>
+                                            <Typography
+                                                {...themeProps.typographyProps}
+                                                classes={{ root: isActive ? classes.typographyActiveRoot : classes.typographyRoot }}
```

Comment:
Shouldn't the `typographyRoot` class always be set?  

I think `typographyActiveRoot` should only be added and not replace `typographyRoot`.  Otherwise all the common non-active-styling and active-styling, which is usually the majority of styling needs to be set twice (e.g. `fontSize: 18`)